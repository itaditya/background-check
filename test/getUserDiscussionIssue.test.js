const getUserDiscussionIssue = require('../lib/github-api/getUserDiscussionIssue')

process.env.APP_ORG_NAME = 'probot-background-check'

test('getUserDiscussionIssue is working', async () => {
  const github = {
    search: { issues: jest.fn() }
  }

  await getUserDiscussionIssue(github, {
    owner: 'itaditya',
    username: 'itaditya'
  })

  expect(github.search.issues.mock.calls).toMatchSnapshot()
})
