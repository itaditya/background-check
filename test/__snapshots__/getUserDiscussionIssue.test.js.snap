// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getUserDiscussionIssue is working 1`] = `
Array [
  Array [
    Object {
      "q": "repo:probot-background-check/itaditya-discussions is:issue is:open in:title itaditya-discussion",
    },
  ],
]
`;
